Objective-C/CBLBlob.mm

```diff
@@ -206,6 +216,11 @@ - (NSData*) content {
         _initialContentStream = nil;
         _content = result;
         _length = _content.length;
+        
+        // update the content and length while loading it
+        [self updateProperties: kLengthMetaProperty value: @(_content.length)];
+        [self updateProperties: kDataMetaProperty value: _content];
+        
```

Comment:
Same as my comment above. We will need to make sure that updating _content and _length are thread-safe.